app/Nova/Travel.php

```diff
@@ -228,6 +228,87 @@ public function fields(Request $request): array
                 ]
             ),
 
+            new Panel(
+                'International Travel',
+                [
+                    Boolean::make('Destination Outside United States', 'is_international')
+                        ->help(
+                            'Check this box if your destination is outside of the United States.'
+                        )
+                        ->hideFromIndex(),
+
+                    Text::make('Justification', 'international_travel_justification')
+                        ->required()
+                        ->rules('required_if:is_international,1', 'nullable')
+                        ->help(
+                            'Please explain how this travel meets essential travel criteria as defined by Georgia Tech.'
+                        )
+                        ->hideFromIndex(),
+
+                    Boolean::make('Export-Controlled Technology', 'export_controlled_technology')
+                        ->required()
+                        ->help(
+                            'Do you plan to take any information or technology that is controlled?'
```

Comment:
Is the assumption that the person completing this knows whether or not something they are planning to take is subject to export controls?

Reply:
Short answer yes. I don't think Nova help text is the place to explain all of this.